cmd/cgr-engine/cgr-engine.go

```diff
@@ -64,6 +64,7 @@ var (
 	cfgDir            = flag.String("config_dir", utils.CONFIG_DIR, "Configuration directory path.")
 	version           = flag.Bool("version", false, "Prints the application version.")
 	pidFile           = flag.String("pid", "", "Write pid file")
+	httpPprofPath     = flag.String("http_pprof_path", "", "http address used for program profiling")
```

Comment:
please rename the flag as httprof_path so we can be consistent with the other two